generator/templates/wire/WireClient.cpp

```diff
@@ -300,11 +384,25 @@ namespace wire {
             self->errorUserdata = userdata;
         }
 
+        // XXX hanlde return command
```

Comment:
It is often used as a "FIXME", and I use it to keep lists of things to do before sending a PR. I should have removed it before committing. Also see https://www.snellman.net/blog/archive/2017-04-17-xxx-fixme/ on the lack of origin for XXX.
Done